pkg/servicemesh/federation/example/config-poc/install.sh

```diff
@@ -51,11 +52,11 @@ echo "Enabling federation mesh2"
 oc create -f import/meshfederation.yaml
 
 echo "Installing istio configuration for mesh1"
-oc create -f export/http/passthrough
-oc create -f export/http/aliased
+# oc create -f export/http/passthrough
+# oc create -f export/http/aliased
 # oc create -f export/http/destinationrule.yaml
-oc create -f export/tcp/passthrough
-oc create -f export/tcp/aliased
+# oc create -f export/tcp/passthrough
+# oc create -f export/tcp/aliased
```

Comment:
Do you want to keep these `PoC` stuff around?

Reply:
I'll eventually delete it.